Dockerfiles/SwarmTest/README.md

```diff
@@ -0,0 +1,17 @@
+# NGINX image for Intel® Clear Containers swarm tests
+
+In order to build the nginx image which is used in our swarm tests,
```

Comment:
Remove "in order" and "which is":
To build the nginx image used in our swarm tests, follow these steps: